src/operator/nn/mkldnn/mkldnn_sum.cc

```diff
@@ -54,41 +54,33 @@ void MKLDNNSum(const mkldnn::memory &arr1,
     in_mem2 = tmp_memory2;
   }
   mkldnn::sum::primitive_desc sum_pd(output_pd, scales, input_pds, CpuEngine::Get()->get_engine());
-  std::unordered_map<int, mkldnn::memory> args = {
+  mkldnn_args_map_t args = {
     { MKLDNN_ARG_MULTIPLE_SRC, *in_mem1 },
     { MKLDNN_ARG_MULTIPLE_SRC + 1, *in_mem2 },
     { MKLDNN_ARG_DST, out },
   };
```

Comment:
Also needs to avoid temporary pairs here?

Reply:
I'm not sure this less efficient than serveral assignment.